packages/cypress/resources/features/refapp-3.x/14-appointments/appointments.feature

```diff
@@ -0,0 +1,15 @@
+Feature: Appointments
+
+  Background:
+    Given the user is logged in
+    And the user arrives on a patient’s summary page
+
+  @appointments
+    Scenario: The users should be able to make an appointment
+      When a user clicks on Appointments tab
+      Then the empty upcoming appointment table should displayed
+      When the user clicks on past appointments section
+      Then the empty past appointment table should displayed
+#  ToDo: Requests an appointment
+#      When a user requests an appointment
+#      Then the request should saved successfully
```

Comment:
Is there any blockers on implementing this?

Reply:
In the appointment schedule forum, there are no data for selecting a service type. So we cannot request appointments for the patients. So this is a blocker for implementing the test further.
ss-
<img width="693" alt="image" src="https://user-images.githubusercontent.com/77311602/188282829-e64c0708-a25d-40ac-9e9a-4e10ad74b836.png">

cc: @jayasanka-sack

Reply:
Could you please raise this issue at the talk?

Reply:
Yeah, sure!